src/services/navigationBar.ts

```diff
@@ -681,6 +682,9 @@ namespace ts.NavigationBar {
                 return isExternalModule(sourceFile)
                     ? `"${escapeString(getBaseFileName(removeFileExtension(normalizePath(sourceFile.fileName))))}"`
                     : "<global>";
+            case SyntaxKind.ExportAssignment:
+                return isExportAssignment(node) && node.isExportEquals ? "export=" : "default";

```

Comment:
Use `InternalSymbolName.ExportEquals` and `InternalSymbolName.Default`  ?

Reply:
Yeah, done. This makes sense - the rest of the file doesn't use this, I can switch things over if you prefer (but for now I've just migrated my changes to use `InternalSymbolName` )